charts/headlamp/tests/test_cases/extra-args.yaml

```diff
@@ -0,0 +1,3 @@
+config:
+  extraArgs:
+  - insecure-ssl
```

Comment:
I know this is just testing the formatting of the template, but the option should start with `-`.